[2018-04-27 05:52:01] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(24): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 05:52:10] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(24): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:02:11] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(28): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:12:24] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(28): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:20:39] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(28): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:45:19] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(28): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:48:10] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:51:05] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(24): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:51:06] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(24): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:51:52] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(57): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-27 06:52:17] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(57): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-28 06:13:12] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:228
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(57): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-28 06:24:50] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php:226
Stack trace:
#0 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 C:\http\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 C:\http\server\public\index.php(57): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 16:06:22] lumen.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:226
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:48:50] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:48:50] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:48:50] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:49:03] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:49:03] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:49:16] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:49:16] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:49:16] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 17:49:37] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
[2018-04-29 21:21:44] lumen.ERROR: Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException in /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php:228
Stack trace:
#0 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(164): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#1 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(413): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(166): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#3 /Users/cailei/http/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(107): Laravel\Lumen\Application->dispatch(NULL)
#4 /Users/cailei/http/server/public/index.php(90): Laravel\Lumen\Application->run()
#5 {main}  
